src/harness/reference_models/geo/census_tract.py

```diff
@@ -31,7 +34,7 @@ def SetCensusTractDirectory(self, census_tract_directory):
       self._census_tract_dir = CONFIG.GetCensusTractsDir()
 
   def GetCensusTract(self, fips_code):
```

Comment:
This is the only required method of the driver I think for the test harness purpose

Reply:
Fixed   as per suggestion.